docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -3,41 +3,70 @@ Datasets
+  For generating a file that contains paths to audio files (which we call as ``scp file``), you can simply use ``find`` bash command as shown below:
 .. code-block:: bash
 
   find $PWD/{data_dir} -iname '*.wav' > path_to_audiofiles.scp
 
+.. note::
+  We expect audio and the corresponding RTTM to have the same base name and the name should be unique (uniq-id).
 
-Preparing Evaluation Dataset
-----------------------------
+oracle_num_speakers:
+  - if number of speakers is known input it or pass null if not known. Accepts int or path to file containing uniq-id with num of speakers of that session 
+
+  Sample line (uniq-id `iaaa` session has 2 unique speakers )::
+    
+    iaaa 2
+
+path2groundtruth_rttm_files [Optional]:
+  - To evaluate the diarizer system with known rttm files, One needs to provide an scp like file (path2groundtruth_rttm_files) for groundtruth label files.
+  use above mentioned ``find`` command to get all reference rttm files (use '*.rttm' as search pattern)
 
-To score with a diarizer model, we need to provide an scp like file for groundtruth label file.
 Each groundtruth label file should be in NIST Rich Transcription Time Marked (RTTM) format. Take one line from a RTTM file for example:
 
 .. code-block:: bash
 
   SPEAKER TS3012d.Mix-Headset 1 331.573 0.671 <NA> <NA> MTD046ID <NA> <NA>
 
 
-Prepraing ORACLE manifest
--------------------------
+oracle_vad_manifest [Optional but suggested for good performace. This avoids FPR and TPR]:
+  - To perform just oracle diarization, that is taking speech activity time stamps from groundtruths or another VAD from wild instead from NeMo VAD output, ``diarizer`` expects an orcale manifest json file that contains paths to audio files with offset for start time and duration of segment.
 
-To perform just oracle diarization, that is taking speech activity time stamps from groundtruths instead from VAD output, ``diarizer`` expects 
-an orcale manifest file that contains paths to audio files with offset for start time and duration of segment.
+To prepare an oracle manifest file, use the script from ``scripts`` folder as shown below:
+
+.. code-block:: bash
 
-To prepare an oracle manifest file, use the helper function from ``speaker_utils`` as shown below:
+  python $NeMo/scripts/ --paths2rttm_files=<path2groundtruth_rttm_files.scp> --paths2audio_files=<paths2audio_files.scp> --manifest_file=<output_oracle_manifest_file.json>
 
-.. code-block:: python
+Here ``paths2audio_files`` and ``path2groundtruth_rttm_files`` are files containing paths to audio files as shown above.
 
-    from nemo.collections.asr.parts.speaker_utils import write_rttm2manifest
+AMI Meeting Corpus
+------------------
+
+The following are the suggested arguments and considerations we took for getting Speaker Error Rate (SER) of 4.1% on AMI Lapel test set corpus:
+  - diarizer.oracle_num_speakers = 4 (since there are exactly 4 speakers per each lapel test set session)
```

Comment:
Misspelling

Reply:
which one? lapel? its correct. Mention the misspelled word if possible that would be easy for me